spec/dummy/config/boot.rb

```diff
@@ -1,13 +1,15 @@
 require 'rubygems'
 
-# The default gemfile is rails4
-gemfile = File.expand_path('../../../../gemfiles/rails4.gemfile', __FILE__)
+# Default to rails7.0.gemfile if BUNDLE_GEMFILE is not set
+default_gemfile = File.expand_path('../../../../gemfiles/rails7.0.gemfile', __FILE__)
```

Comment:
Minor simplification suggestion while we’re at it.

```suggestion
default_gemfile = File.expand_path('../../../gemfiles/rails7.0.gemfile', __dir__)
```